condor_watch_q.py

```diff
@@ -630,12 +658,12 @@ def __str__(self):
     def ordered(cls):
         return (
             cls.REMOVED,
-            cls.HELD,
             cls.SUSPENDED,
-            cls.IDLE,
+            cls.COMPLETED,
```

Comment:
@bbockelm I prefer the ordering I had originally, because it flows from left to right. Thoughts?

Reply:
I have no opinions here other than being consistent with the `condor_q` output.